docs/CodeDoc/Atc.Network/Atc.Network.md

```diff
@@ -176,6 +176,38 @@ ConnectionStateEventArgs.
 
 
 
+<br />
+
+## IPStatusExtensions
+Provides extension methods for the `System.Net.NetworkInformation.IPStatus` enumeration.
+><b>Remarks:</b> This static class extends the `System.Net.NetworkInformation.IPStatus` enumeration, allowing for easy retrieval of localized string descriptions for each IP status value. This can be particularly useful for displaying user-friendly status messages in applications that perform network operations and diagnostics.
+
+>```csharp
+>public static class IPStatusExtensions
+>```
+
+### Static Methods
+
+#### GetLocalizedDescription
+>```csharp
+>string GetLocalizedDescription(this IPStatus ipStatus)
+>```
+><b>Summary:</b> Retrieves a localized description for a specified `System.Net.NetworkInformation.IPStatus` value.
+>
+><b>Parameters:</b><br>
+>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`ipStatus`&nbsp;&nbsp;-&nbsp;&nbsp;The  value for which a localized description is needed.<br />
```

Comment:
Superfluous whitespace

Reply:
Bug in markdown-generator for "see cref", this will be fixed in Atc first.